tests/cbmc/proofs/Makefile.common

```diff
@@ -259,6 +259,11 @@ CBMC_FLAG_FLUSH ?= --flush
 
 CBMCFLAGS += $(CBMC_FLAG_FLUSH)
 
+# CBMC 6.0.0 enables all standard checks by default, which can make coverage analysis
+# very slow. See https://github.com/diffblue/cbmc/issues/8389
+# For now, we disable these checks when generating coverage info.
+COVERFLAGS ?= --no-standard-checks
```

Comment:
What checks do we lose here? Why is this safe?

Reply:
This flag only applies to the "Generate Coverage Report" phase of the run. The actual generation and checking of the proofs is unaffected. CBMC team advised this was the right thing to do.

Reply:
Further to what Rod said: we check those properties, and without the above option we would _assume_ those properties to hold during coverage checking. This isn't actually necessary as we have proved them in the checking phase, so we can safely skip on that during coverage checking and get better performance. We will likely fix this on the CBMC side and effectively make that the default.